src/buckets/components/createBucketForm/UpdateBucketOverlay.tsx

```diff
@@ -70,10 +79,16 @@ const UpdateBucketOverlay: FunctionComponent<Props> = ({
         handleClose()
         return
       }
+
       setBucketDraft(resp.data as OwnBucket)
 
       setSchemaType(resp.data.schemaType)
 
+      if ('explicit' === resp.data.schemaType) {
+        const schema = await getSchema(bucketID)
+        setMeasurementSchemaList(schema)
```

Comment:
this is another place to put `event` calls for measuring usage of this feature

Reply:
this is just getting the data when the user is viewing or editing a bucket.  so not actually an event.

Reply:
i *did* just check and there is no event fired when the user clicks the button to view the settings, so I added one there  ( event('bucket.view.settings')